### ISSUE TITLE
How to support targeted attack? 

### ISSUE THREAD

**Issue by machanic** [satyanshukla/bayes_attack#1]
How to modify this code to support targeted attack using target class label?

**Comment by satyanshukla** [comment]
So, for targeted attacks in case of hard label, replace this 
`f = torch.where(index == y0, torch.ones_like(index), torch.zeros_like(index)).float()`
by
`f = torch.where(index == y_target, torch.zeros_like(index), torch.ones_like(index)).float()`
